server/models/request.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+from sqlalchemy import (
+    Column, String,
+    Integer, ForeignKey,
+    Text, Boolean
+    )
+from sqlalchemy.orm import relationship, validates
+from models.base_model import Base, BaseModelMixin
+
+from models.recipe import Recipe
+from models.user import User
+
+class Request(Base, BaseModelMixin):
+
+    __tablename__ = 'request'
+
+    id = Column(Integer, primary_key=True)
+    chef_id = Column(Integer, ForeignKey('user.id'))
+    user_id = Column(Integer, ForeignKey('user.id'))
+    recipe_id = Column(Integer, ForeignKey('recipe.id'))
+    created_date_time = Column(DateTime, default=datetime.now)
+
+
+    accepted = Column(db.Boolean) #indicates request has been made by the user/customer and accepted by the chef
+    request_fulfilled = Column(db.Boolean) #indicates request has been paid for by the user/customer
+    quantity = Column(db.Integer) 
```

Comment:
What's the thought process behind having a quantity field? If you have a relationship field to the recipe model you can use something like in this github link ( https://gist.github.com/hest/8798884 ) you *should* be able to just use `self.recipes` instead of performing an actual query; I may be wrong but it's worth a try. Use the part that has the comment `# Fast`. you can rename the function as 'recipe_count' and have it be a `@property` so you can call it like this `amount_of_recipes = request.recipe_count`